Formula/d/djview4.rb

```diff
@@ -32,18 +31,13 @@ class Djview4 < Formula
   depends_on "pkgconf" => :build
   depends_on "djvulibre"
   depends_on "libtiff"
-  depends_on "qt@5"
-
-  # Fix QT detection when multiple Xcode installations are present.
-  # Submitted upstream: https://sourceforge.net/p/djvu/patches/44/
-  patch :DATA
+  depends_on "qt"
 
   def install
-    system "autoreconf", "--force", "--install", "--verbose"
-
-    system "./configure", "--with-x=no",
+    system "./autogen.sh", "--with-x=no",
                           "--disable-nsdejavu",
-                          "--disable-desktopfiles",
+                          "--#{if OS.mac? then 'disable' else 'enable' end}-desktopfiles",
+                          "--#{if OS.mac? then 'enable' else 'disable' end}-mac",
```

Comment:
```suggestion
                          "--#{OS.mac? ? 'disable' : 'enable'}-desktopfiles",
                          "--#{OS.mac? ? 'enable' : 'disable'}-mac",
```
Use the ternary operator here instead.